pyrolite/data/models/sandstones/config_herron.json

```diff
@@ -0,0 +1,13 @@
+{"name": "Herron", "axes":{"x":"log(SiO2/Al2O3)","y":"log(Fe2O3/K2O)"}, 
```

Comment:
To avoid potential ambiguity here, I'd suggest using either e.g. `ln(SiO2/Al2O3)` or `log10(SiO2/Al2O3)` as is appropriate.